lumicks/pylake/kymotracker/detail/msd_estimation.py

```diff
@@ -630,8 +630,22 @@ def calculate_localization_error(frame_lags, msd):
         frame lags to include.
     msd : array_like
```

Comment:
As you changed all the other docstring types, probably this one, too?
```suggestion
    msd : np.ndarray
```

Reply:
I think `array_like` is good enough here. It produces the exact same result with any `array_like`.

In the return argument documentation we returned an `ndarray` (there we know what we return).
And in the other function args, we immediately applied an operation on them without invoking `np.asarray` first (which would result in incorrect behaviour and therefore the input there had to be `np.ndarray`).